actions/class.Items.php

```diff
@@ -126,7 +126,7 @@ public function moveInstance()
     /**
      * overwrite the parent moveAllInstances to add the requiresRight only in Items
      * @see tao_actions_TaoModule::moveResource()
-     * @requiresRight uri WRITE
+     * @requiresRight classUri WRITE
```

Comment:
Does it apply the ACL after this change? If you add/remove user permissions for a class, does it display/hide the button?

Do we check permission on destination folder?

Reply:
Yes. When you set acl for a class it will be disabled on a list. When user will click on it he will not see any actions. When you are able to write on class and you reach to screen when you can chose target class, classes that are not allowed will be disabled so you cannot choose it.